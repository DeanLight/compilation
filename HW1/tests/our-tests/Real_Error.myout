1 REAL 1.0
1 REAL .1
