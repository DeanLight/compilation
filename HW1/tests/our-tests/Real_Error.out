1 REAL 1.0
1 REAL .1
2 REAL 1.
2 REAL .1
3 REAL 1.
Error .
